saritasa_invocations/k8s.py

```diff
@@ -0,0 +1,222 @@
+import invoke
+
+from . import _config, printing
+
+
+def get_current_env_config_from_context(
+    context: invoke.Context,
+) -> _config.K8SSettings:
+    """Return current environment data class based on current cluster."""
+    run_result = context.run(
+        "kubectl config current-context",
+        echo=False,
+        hide="out",
+    )
+    if not run_result:
+        raise invoke.Exit(
+            code=1,
+            message="Unexpected error, make sure to run inv `k8s.set_context`",
+        )
+    current_cluster = run_result.stdout.splitlines()[0]
+
+    run_result = context.run(
+        "kubectl config view --minify | grep namespace | awk '{print $2}'",
+        echo=False,
+        hide="out",
+    )
+    if not run_result:
+        raise invoke.Exit(
+            code=1,
+            message="Unexpected error, make sure to run `inv k8s.set_context`",
+        )
+    current_namespace = run_result.stdout.splitlines()[0]
+
+    for env in _config._K8S_CONFIGS.values():
+        if (
+            env.cluster == current_cluster
+            and current_namespace in env.namespace
+        ):
+            return env
+
+    raise invoke.Exit(
+        code=1,
+        message=(
+            f"Environment data class for the cluster `{current_cluster}`"
+            f" and namespace `{current_namespace}` doesn't exits."
+        ),
+    )
+
+
+def get_environment(
+    context: invoke.Context,
+    env_name: str,
+) -> _config.K8SSettings:
+    """Get environment by its name."""
+    config = _config.Config.from_context(context)
+    environment = config.k8s_configs.get(env_name)
+    if not environment:
+        raise invoke.Exit(
+            code=1,
+            message=(
+                f"Data class for `{env_name}` environment not found. "
+                "Available environments: "
+                f"{', '.join(config.k8s_configs.keys())}"
+            ),
+        )
+    return environment
+
+
+def get_pod_cmd(
+    context: invoke.Context,
+    component: str,
+) -> str:
+    """Get command for getting exact pod."""
+    config = get_current_env_config_from_context(context)
+    return (
+        "kubectl get pods "
+        f"--selector {config.pod_label}={component} "
+        "--no-headers --output jsonpath='{.items[*].metadata.name}'"
+    )
+
+
+@invoke.task
+def set_context(context: invoke.Context, env: str = "") -> None:
+    """Set k8s context to current project."""
+    printing.print_success("Setting context for k8s")
+    config = _config.Config.from_context(context)
+    env = env or config.default_k8s_env
+    environment = get_environment(context, env)
+    context.run(f"kubectl config use-context {environment.cluster}")
+    context.run(
+        f"kubectl config set-context "
+        f"--current --namespace={environment.namespace}",
+    )
```

Comment:
One code style like in `login`
```suggestion
        f"kubectl config set-context"
        f" --current --namespace={environment.namespace}",
    )
```